dashboard/test/controllers/courses_controller_test.rb

```diff
@@ -853,4 +853,14 @@ class CachedQueryCounts < ActionController::TestCase
     assert_equal 2, response_body.length
     assert_equal(['All Resources', 'All Standards'], response_body.map {|r| r['name']})
   end
+
+  test "all: shows all courses " do
+    Rails.application.config.stubs(:levelbuilder_mode).returns true
+
+    sign_in(create(:levelbuilder))
+    get :all
+    assert_response :success
+    refute_nil assigns(:courses)
+    assert_equal UnitGroup.all, assigns(:courses)
+  end
```

Comment:
please also test that the route is not visible to ordinary users or signed out users. you can write oneliners to do this via `test_user_gets_response_for`